quickstart/python/app.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    target = os.environ.get('TARGET', 'World')
```

Comment:
I don't see how TARGET ever gets set, so is this needed?

Reply:
No it isn't needed, but I've left it in all the examples for now just to make the code slightly more interesting.